### PR TITLE
docs: Add HOWTO in Turkish

### PR DESCRIPTION
**Pull request by Shubuo**
## What does this PR do?
**Add resource(s)** | Remove resource(s) | Add info | Improve repo

### Description
For hacktoberfest purpose

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
**TIP :writing_hand::** It's a good practice to **fill Pull Request fields and check tasklist contracts**. It helps a bit we can understand your contribution better :hugs:. and then, increase maintainers productivity :rocket:

**Review by davorpa — COMMENTED** [review]
translated twice: en-tr

**Comment by eshellman** [comment]
Anything else?  Crosslinks will be reorganized once this is merged

File: HOWTO-tr.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* [Youtube - Yeni Başlayanlar İçin Github Eğitimi](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Bir GitHub Repo Nasıl Çatallanır ve Bir Çekme Talebi Nasıl Gönderilir](https://www.youtube.
+com/watch?v=G1I3HF4YWEw)
+* [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Soru sormaktan çekinmeyin; her katılımcı ilk bir PR ile başladı. Binincimiz olabilirsin!
+
+Deneyimli bir açık kaynak katılımcısı olsanız bile, sizi rahatsız edebilecek şeyler var. PR'nizi gönderdikten sonra,
+GitHub Eylemleri, genellikle boşluk veya alfabetik sıralama ile ilgili küçük sorunlar bularak bir linter çalıştırır. Yeşil bir düğme alırsanız, her şey gözden geçirilmeye hazırdır, ancak değilse, linter'in neyi sevmediğini bulmak için başarısız olan kontrolün altındaki "Ayrıntılar" ı tıklayın. Sorunu düzeltin ve PR'nize bir taahhüt ekleyin.
```
Comment (davorpa) [inline HOWTO-tr.md:17]: Avoid translate trademarks / product names / UI elements

```suggestion
GitHub Actions, genellikle boşluk veya alfabetik sıralama ile ilgili küçük sorunlar bularak bir linter çalıştırır. Yeşil bir düğme alırsanız, her şey gözden geçirilmeye hazırdır, ancak değilse, linter'in neyi sevmediğini bulmak için başarısız olan kontrolün altındaki "Details" ı tıklayın. Sorunu düzeltin ve PR'nize bir taahhüt ekleyin.
```

File: HOWTO-tr.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Diğer dillerde okumak için: [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md) , [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [ Deutsch](HOWTO-de.md), [Bahasa Endonezya](HOWTO-id.md), [Tiếng Việt](HOWTO-vi.md), [عربي](HOWTO-ar.md).
+Serbest Programlama Kitaplarına Hoş Geldiniz! Github'da ilk çekme talebini yapanlardan olsanız bile Katkıda bulunmak için yeni gelenleri memnuniyetle karşılıyoruz. Eğer onlardan biriyseniz, işte size yardımcı olabilecek bazı kaynaklar:
```
Comment (davorpa) [inline HOWTO-tr.md:2]: - Recover repo name
- Homogenize trademark
- Introduction of PR acronym

```suggestion
Free-Programming-Books Hoş Geldiniz! GitHub'da ilk Çekme Talebini (PR) yapanlardan olsanız bile Katkıda bulunmak için yeni gelenleri memnuniyetle karşılıyoruz. Eğer onlardan biriyseniz, işte size yardımcı olabilecek bazı kaynaklar:
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

File: HOWTO-tr.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+com/watch?v=G1I3HF4YWEw)
+* [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Soru sormaktan çekinmeyin; her katılımcı ilk bir PR ile başladı. Binincimiz olabilirsin!
+
+Deneyimli bir açık kaynak katılımcısı olsanız bile, sizi rahatsız edebilecek şeyler var. PR'nizi gönderdikten sonra,
+GitHub Eylemleri, genellikle boşluk veya alfabetik sıralama ile ilgili küçük sorunlar bularak bir linter çalıştırır. Yeşil bir düğme alırsanız, her şey gözden geçirilmeye hazırdır, ancak değilse, linter'in neyi sevmediğini bulmak için başarısız olan kontrolün altındaki "Ayrıntılar" ı tıklayın. Sorunu düzeltin ve PR'nize bir taahhüt ekleyin.
+
+Nihayet, Eklemek istediğiniz kaynağın Serbest Programlama Kitapları için uygun olduğundan emin değilseniz, [CONTRIBUTING](CONTRIBUTING.md) bölümündeki yönergeleri okuyun.
```
Comment (davorpa) [inline HOWTO-tr.md:19]: Recover repo name

```suggestion
Nihayet, Eklemek istediğiniz kaynağın Free-Programming-Books için uygun olduğundan emin değilseniz, [CONTRIBUTING](CONTRIBUTING.md) bölümündeki yönergeleri okuyun.
```

**Review by davorpa — COMMENTED** [review]
Yes, there are some issues to solve.
- Markdown format bugs
- Some Translation avoidings
- Notes for in which language is the content of outlinks

**Comment by davorpa** [comment]
Review completed at #6618

**Pull request merged**